nodes/src/nodes/memory_internal/IInstance.py

```
@@ -6,34 +6,123 @@
     """Pipeline instance for the memory_internal tool node."""
 
     IGlobal: IGlobal
-    driver: MemoryDriver = None
+    _store: MemoryStore | None = None
 
     def beginInstance(self) -> None:
-        """Create the private MemoryDriver when the pipe is instantiated."""
-        self.driver = MemoryDriver()
+        self._store = MemoryStore()
 
     def open(self, _obj: Any) -> None:
```

Ruff (0.15.7)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `_obj`

(ANN401)